<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkiRent - user</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="countryCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cityCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phoneEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Surname:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surnameEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Document:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="documentTypeCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Document number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="documentNumberEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
